### ISSUE TITLE
RTPS inflation documentation and initialization code need updating

### ISSUE THREAD

**Issue by nancycollins** [NCAR/DART#177]
the inflation documentation needs to discuss the RTPS option -- how it works, what values are needed, what values are written out, and how the namelist settings are used in the algorithm.

the adaptive inflation module code should error out if the namelist settings for: "read initial values from a file rather than the namelist", damping, and deterministic don't match what the code is going to use.  currently these are warning messages only and the namelist values are simply reset.  these warning messages could easily be missed in the log.  forcing the user to set them to be consistent with how they are used in the code would be less prone to misunderstanding.

**Comment by mgharamti** [comment]
Yes, totally agree! I'll work on the RTPS documentation starting from tomorrow.

**Comment by hkershaw-brown** [comment]
Moha's docs added to master in https://github.com/NCAR/DART/pull/179

**Comment by nancycollins** [comment]
thanks moha!

**Comment by timhoar** [comment]
The part about the adaptive_inflate_mod.f90 erroring out if someone has the namelist set to read from file is still outstanding.
The current code reads:

```
! RTPS needs a single parameter from namelist: inf_initial(2).  
! Do not read in any files.  Also, no damping.  
! Warn the user if they try to set different values in the namelist.
if (inf_flavor(POSTERIOR_INF) == RELAXATION_TO_PRIOR_SPREAD) then
   if (inf_initial_from_restart(POSTERIOR_INF) .or. inf_sd_initial_from_restart(POSTERIOR_INF)) &
      call error_handler(E_MSG, 'validate_inflate_options:', &
         'RTPS inflation (type 4) overrides posterior inflation restart file with value in namelist', &
         text2='posterior inflation standard deviation value not used in RTPS')
```

**Comment by nancycollins** [comment]
sorry - forgot about the code changes!  that one, plus in the code that follows there are 2 other overrides that need to error out as well.   we need a different message text that doesn't say 'override' since it's an error now.  something like 'X must be Y for RTPS inflation' .

**Comment by nancycollins** [comment]
sorry - wrong button!!
